src/OrbitGl/GraphTrack.cpp

```diff
@@ -37,24 +37,29 @@ GraphTrack<Dimension>::GraphTrack(CaptureViewElement* parent,
     : Track(parent, timeline_info, viewport, layout, module_manager, capture_data),
       series_{series_names, series_value_decimal_digits, std::move(series_value_units)} {}
 
+template <size_t Dimension>
+bool GraphTrack<Dimension>::HasLegend() const {
+  return !IsCollapsed() && Dimension > 1;
+}
+
 template <size_t Dimension>
 float GraphTrack<Dimension>::GetHeight() const {
-  // Top content margin is counted twice because it it inserted before and after the legend
-  float height = layout_->GetTrackTabHeight() + layout_->GetTrackContentTopMargin() * 2 +
-                 GetLegendHeight() + GetGraphContentHeight() +
-                 layout_->GetTrackContentBottomMargin();
-  return height;
+  // Top content margin is counted twice when there are legends because it is inserted above and
+  // below the legend.
+  float legend_height_with_margins =
```

Comment:
I don't like this name. 

I would rather prefer: "legend_height_and_margins" and include bottom_margin, or something like "height_above_content" as you suggested in the chat a few days ago.

Reply:
We can have an offline discussion on the naming if you don't mind